02_configure_host.sh

```diff
@@ -121,23 +125,44 @@ fi
 
 # Add firewall rules to ensure the IPA ramdisk can reach httpd, Ironic and the Inspector API on the host
 for port in 80 5050 6385 ; do
-    if ! sudo iptables -C INPUT -i provisioning -p tcp -m tcp --dport $port -j ACCEPT > /dev/null 2>&1; then
-        sudo iptables -I INPUT -i provisioning -p tcp -m tcp --dport $port -j ACCEPT
+    if [ "${RHEL8}" = "y" ] ; then
+        sudo firewall-cmd --zone=libvirt --add-port=${port}/tcp
+        sudo firewall-cmd --zone=libvirt --add-port=${port}/tcp --permanent
+    else
+        if ! sudo iptables -C INPUT -i provisioning -p tcp -m tcp --dport $port -j ACCEPT > /dev/null 2>&1; then
+            sudo iptables -I INPUT -i provisioning -p tcp -m tcp --dport $port -j ACCEPT
+        fi
     fi
 done
 
 # Allow ipmi to the virtual bmc processes that we just started
-if ! sudo iptables -C INPUT -i baremetal -p udp -m udp --dport 6230:6235 -j ACCEPT 2>/dev/null ; then
-    sudo iptables -I INPUT -i baremetal -p udp -m udp --dport 6230:6235 -j ACCEPT
+if [ "${RHEL8}" = "y" ] ; then
+    sudo firewall-cmd --zone=libvirt --add-port=6230-6235/udp
+    sudo firewall-cmd --zone=libvirt --add-port=6230-6235/udp --permanent
+else
+    if ! sudo iptables -C INPUT -i baremetal -p udp -m udp --dport 6230:6235 -j ACCEPT 2>/dev/null ; then
+        sudo iptables -I INPUT -i baremetal -p udp -m udp --dport 6230:6235 -j ACCEPT
+    fi
 fi
 
 #Allow access to dhcp and tftp server for pxeboot
 for port in 67 69 ; do
-    if ! sudo iptables -C INPUT -i provisioning -p udp --dport $port -j ACCEPT 2>/dev/null ; then
-        sudo iptables -I INPUT -i provisioning -p udp --dport $port -j ACCEPT
+    if [ "${RHEL8}" = "y" ] ; then
+        sudo firewall-cmd --zone=libvirt --add-port=${port}/udp
+        sudo firewall-cmd --zone=libvirt --add-port=${port}/udp --permanent
+    else
+        if ! sudo iptables -C INPUT -i provisioning -p udp --dport $port -j ACCEPT 2>/dev/null ; then
+            sudo iptables -I INPUT -i provisioning -p udp --dport $port -j ACCEPT
+        fi
     fi
 done
 
+# mDNS
+if [ "${RHEL8}" = "y" ] ; then
```

Comment:
In theory, we should be doing it for RHEL7 as well. However, I suspect our firewall is wide open by default, and all these `iptables` lines are not needed..

Reply:
Yeah, I just didn't want to break centos7 envs just yet.  I figured I'd get it working on rhel8 with firewall-cmd and then come back later and test using that on centos7.